project-base/storefront/components/Pages/BlogCategory/BlogCategoryArticlesWrapper.tsx

```diff
@@ -35,8 +37,14 @@ export const BlogCategoryArticlesWrapper: FC<BlogCategoryArticlesWrapperProps> =
 
     return (
         <>
-            {!!mappedArticles?.length && !areBlogCategoryArticlesFetching ? (
-                <BlogArticlesList blogArticles={mappedArticles} />
+            {!areBlogCategoryArticlesFetching ? (
+                mappedArticles?.length ? (
+                    <BlogArticlesList blogArticles={mappedArticles} />
+                ) : (
+                    <div className="flex flex-col gap-10">
+                        <div>{t('Sorry, there are no articles in this category at the moment.')}</div>
+                    </div>
+                )
             ) : (
                 <div className="flex flex-col gap-10">
                     {createEmptyArray(DEFAULT_PAGE_SIZE).map((_, index) => (
```

Comment:
I still have two things to consider here:

Sure, here is the translation:

1. You are using a ternary operator where the condition has a negated boolean (`!isFetching ? something : somethingElse` instead of `isFetching ? somethingElse : something`)
2. You are using a nested ternary operator

Reply:
rewritten in [ebb3e3d](https://github.com/shopsys/shopsys/pull/3292/commits/ebb3e3d3e0f7afc10f6c41e5b6ccc96b5c5fda0c) :slightly_smiling_face: